api/controllers/project.js

```diff
@@ -592,12 +606,13 @@ exports.protectedAddPins = async function (args, res, next) {
     Utils.recordAction('Add', 'Pin', args.swagger.params.auth_payload.preferred_username, objId);
     return Actions.sendResponse(res, 200, doc);
   } else {
-    defaultLog.info("Couldn't find that object!");
+    defaultLog.info("Error adding project pins");
```

Comment:
so this is an info log, but you have the word Error in the text.. I'm confused. again context would be great.